ntbs-service/Startup.cs

```diff
@@ -46,6 +50,22 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            services.AddHangfire(config => {
+                config.SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
+                    .UseSimpleAssemblyNameTypeSerializer()
+                    .UseRecommendedSerializerSettings()
+                    .UseSqlServerStorage(Configuration.GetConnectionString("hangfireContext"), new SqlServerStorageOptions
```

Comment:
Are the tables hangfire creates prefix with something that makes them very obviously to do with hangfire? I think ideally we would just use the ntbsContext and avoid adding another DB, as it will be a bit of a pain to add databases ad-hoc in live. Do you forsee any problems with just using `ntbsContext` here @ffarmanov ?